tests/func/test_update.py

```diff
@@ -1,9 +1,12 @@
 import filecmp
 import os
 import shutil
+import pytest
 
+from dvc.exceptions import UpdateWithRevNotPossibleError
 from dvc.external_repo import clean_repos
 from dvc.repo import Repo
+from dvc.utils.compat import fspath
```

Comment:
```suggestion
from dvc.compat import fspath
```
utils.compat was deleted recently. This is the reason why the test is failing.

Reply:
👍 interesting, it was working on my machine...